samcli/commands/check/resources/lambda_function_pricing.py

```diff
@@ -52,7 +52,10 @@ def __init__(self):
         self._max_memory_amount = 10000
         self._max_duration = 900000
 
-    def ask_lambda_function_questions(self) -> None:
+    def ask_questions(self) -> None:
+        self._ask_lambda_function_questions()
```

Comment:
This looks redundant to me. The `_ask_lambda_function_questions` method can be implemented directly under `ask_questions`. Is there any reason to create a standalone `_ask_lambda_function_questions` method?

Reply:
You're right. This is a bit redundant. `_ask_lambda_function_questions` should just be directly implemented under `ask_questions`